src/MicrosoftTeams.ts

```diff
@@ -49,6 +49,176 @@ namespace microsoftTeams {
         args?: any[]; // tslint:disable-line:no-any The args here are a passthrough from OnMessage where we do receive any[]
     }
 
+     /**
+     * Namespace to send message to SDK from native apps.
```

Comment:
This comment should be more generic and mention about the use in frame-less scenario. And you should have another comment before postMessage which mentions about its use for sending messages.

Reply:
done